bciers/apps/administration/tests/components/operators/OperatorDetailsPage.test.tsx

```diff
@@ -0,0 +1,76 @@
+import { render, screen } from "@testing-library/react";
+import { getOperator } from "./mocks";
+import OperatorDetailsPage from "@/administration/app/components/operators/OperatorDetailsPage";
+import { expect } from "vitest";
+
+vi.mock("@bciers/actions", () => ({
+  fetchDashboardData: vi.fn(() => [
```

Comment:
Could we pull this from the fixture instead? If we change any of the tile data, this test will always be testing the data in this file instead of whatever new things we've added

Reply:
I believe this responsibility goes beyond this test; it should be addressed in e2e tests because we never fetch real data in our Vitests and always mock the data.